Analyze case with vlinder.ipynb

```diff
@@ -638,31 +519,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Folder 'json' already exists. \n",
-      "Folder 'csv' already exists. \n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "output_path = Path.cwd() / 'data' / name \n",
     "case.transform(output_path, \"json\")\n",
     "# case.transform(output_path, \"xlsx\")\n",
```

Comment:
waarom deze uitgecomment?

Reply:
Deze was al uitgecomment in de oorspronkelijke analyze_case.ipynb notebook, ik denk omdat bij het inlezen al een xlsx format is gebruikt (bij case = TheResponsibleBusinessSimulator(path, file_format, name)).